dataprep/eda/dtypes.py

```diff
@@ -233,3 +255,27 @@ def is_pandas_categorical(dtype: Any) -> bool:
     Detect if a dtype is categorical and from pandas.
     """
     return any(isinstance(dtype, c) for c in CATEGORICAL_PANDAS_DTYPES)
+
+
+def drop_null(
+    var: Union[dd.Series, pd.DataFrame, dd.DataFrame]
+) -> Union[pd.Series, dd.Series, pd.DataFrame, dd.DataFrame]:
+    """
+    Drop the null values (specified in NULL_VALUES) from a series or DataFrame
+    """
+
+    if isinstance(var, (pd.Series, dd.Series)):
+        if is_datetime(var.dtype):
+            return var.dropna()
+        return var[~var.isin(NULL_VALUES)]
+
+    elif isinstance(var, (pd.DataFrame, dd.DataFrame)):
+        df = var
+        for values in df.columns:
+            if is_datetime(df[values].dtype):
+                df = df.dropna(subset=[values])
+            else:
+                df = df[~df[values].isin(NULL_VALUES)]
+        return df
+
+    return None
```

Comment:
It would be better to make the error explicit by raising an exception.

Reply:
Hi Young, I added a ValueError with a custom message. For setting `DATAPREP_SKIP_CREDENTIAL_TESTS` to 1, in what file is this variable? test_integration.py?

Reply:
You can just set it in your environment variables